<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wikipedia Race</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="graphLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>501</width>
      <height>381</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>138</width>
      <height>188</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="startingPageLabel">
         <property name="text">
          <string>Starting Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="startingPageLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="endingPageLabel">
         <property name="text">
          <string>Ending Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="endingPageLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="algorithmLabel">
         <property name="text">
          <string>Select Algorithm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="algorithmComboBox">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Breadth-First Search</string>
         </property>
         <item>
          <property name="text">
           <string>Breadth-First Search</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Depth-First Search</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dijkstra's Algorithm</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="solveButton">
       <property name="text">
        <string>Solve</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWikipedia_Game_Solver">
    <property name="title">
     <string>Wikipedia Game Solver</string>
    </property>
   </widget>
   <addaction name="menuWikipedia_Game_Solver"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
